config/resources/domain.lisp

```diff
@@ -12,7 +12,7 @@
   "when non-nil, all paginated listings will contain the number
    of responses in the result object's meta.")
 (defparameter *max-group-sorted-properties* nil)
-(defparameter sparql:*query-log-types* nil)
+;; (defparameter sparql:*query-log-types* nil)
```

Comment:
Guessing you wanted to log stuff, re-enable

Reply:
yeah, I was having struggles with mu-cl-resource but that should not have been committed here, I will revert this

Reply:
re-enabled in [68f7d89](https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/381/commits/68f7d897f348eafb7ff105da780498a739a29ded)